### ISSUE TITLE
Unreadable data

### ISSUE THREAD

**Issue by rliessum** [fvdm/nodejs-tk102#35]
Got a other TK102 fake. What can I deliver to decode this? (packet trace? other logging?)
Looks like a homebrew sockets implementation to me...

Event: data
'(087076826183BP00HSO)'

Event: fail
{ Error: Cannot parse GPS data from device
    at Socket.<anonymous> (/var/www/tk102/node_modules/tk102/tk102.js:152:17)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:547:20)
  reason: 'Cannot parse GPS data from device',
  input: '(087076826183BP00HSO)',
  connection: 
   { address: '0.0.0.0',
     family: 'IPv4',
     port: 5000,
     remoteAddress: 'x.x.x.x',
     remotePort: 38267 } }

**Comment by fvdm** [comment]
I’m no longer updating device support for this package. Even the fakes from the same source have completely different output strings and commands. After wasting more money on 4 different models in order to keep this package current I decided to stay away from closed source trackers. All devices have already died from bad soldering inside anyway.

Your string looks like a heartbeat message where the first number is probably the serial number of the device. It may be listening for commands in order to start sending meaningful data.

The TK102 device I used at first could be watched by only receiving the messages. Setup a TCP server in Node and compare the messages to the current date/time and geolocation of a smartphone. Pretty experimental stuff. Just listen for messages, see what changes when you send different sms commands, and maybe try sending something in return.

I’m sorry I can’t directly help you with your specific model. If I had the device I would start with listening and gradually send back different kinds of commands that are common for other models and see what happens.
